### PR TITLE
2 login feature

### PR DESCRIPTION
**Pull request by kemalmao19**
add functionally login feature

### CODE REVIEW

File: src/app/(auth)/login/page.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Login } from '@/components/auth/components/Login'
+import React from 'react'
+
+export default function Page() {
+  return (
+    <>
```
Comment (indrazm) [inline src/app/(auth)/login/page.js:6]: You don't need a fragment <>

File: src/components/auth/components/Icons/PasswdIcon.jsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React from "react";
+export const PasswdIcon = (props) => (
```
Comment (indrazm) [inline src/components/auth/components/Icons/PasswdIcon.jsx:2]: It doesn't need any props

File: src/components/auth/components/Login.jsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+import {MailIcon} from './Icons/MailIcon';
+import { PasswdIcon } from './Icons/PasswdIcon';
+import { useLogin } from '../hooks/useLogin';
+
+export const Login = () => {
+
+  const {isLoading, handleChange, handleSubmitButton} = useLogin();
+
+  return (
+    <main className='flex flex-col justify-between items-center py-5' style={{height:"100vh"}}>
```
Comment (indrazm) [inline src/components/auth/components/Login.jsx:14]: change `style={{ height: "100vh"}}` to `h-screen` class.

**Review by indrazm — CHANGES_REQUESTED** [review]
Please change. 
Many Thanks.

**Review by indrazm — APPROVED** [review]
lgtm.

**Review by agungagriza — APPROVED** [review]

**Pull request merged**